openedx/features/course_experience/__init__.py

```diff
@@ -58,3 +59,15 @@ def course_home_url_name(course_key):
         return 'openedx.course_experience.course_home'
     else:
         return 'info'
+
+
+class CourseHomeMessages(UserMessageCollection):
```

Comment:
I'd be a little hesitant about calling them course welcome messages, since the user could be nearly at the end of the course and still see these messages when he/she is not yet logged in. Open to switching it to CourseHomeMainMessages, but feel that CourseHomeMessages is descriptive enough for the time being. If we ever add new messages to the home page we can always rework the name.